riptide-failsafe/src/test/java/org/zalando/riptide/failsafe/MockWebServerUtil.java

```diff
@@ -0,0 +1,50 @@
+package org.zalando.riptide.failsafe;
```

Comment:
I've collected common util methods in `MockWebServerUtil` class, currently this class is copied to the every module where I used `okhttp3.mockwebserver.MockWebServer` (the same approach is used for the existing class `MockSetup`). We can avoid copying and pasting it by introducing new project module that will be used only with test scope but I'm not sure if it is worths investing time into

Reply:
I would recommend to skip extracting separate module, totally fine with copying paste in tests